component/service/src/main/java/org/exoplatform/social/rest/impl/user/UserRestResourcesV1.java

```diff
@@ -1377,17 +1378,10 @@ private String importUser(UserImportResultEntity userImportResultEntity,
         LOG.error("Failure to retrieve portal config", e);
       }
       passwordRecoveryService.sendOnboardingEmail(user, locale, url);
+      if (profileEntity != null) {
+        profileEntity.setEnrollmentDate(String.valueOf(Calendar.getInstance().getTimeInMillis()));
+      }
```

Comment:
This should be in sendOnboardEmail method, so that, you can check if the sent is ok or KO

Reply:
I can't update profile inside sendOnboardEmail method , I added a check if the sent is ok or not